### DISCUSSION TITLE
Tor Support

### DISCUSSION THREAD

**Discussion by resignedScientist** [isontheline/pro.webssh.net#648] [Ideas]
I want to be able to connect to my server with the tor network.

If you want to stay anonymous or are not able to open a port for your server, the Tor network would be very useful!

**Comment by isontheline** [comment]
Hello @P1xelfehler 👋

TOR implementation' issue is tracked here : https://github.com/isontheline/pro.webssh.net/issues/362
And in this discussion : https://github.com/isontheline/pro.webssh.net/discussions/353

It's a hard thing to implement 😬
